agentai/conversation.py

```diff
@@ -13,8 +13,11 @@ class Message(BaseModel):
 
 
 class Conversation:
-    def __init__(self, history: List[Message] = [], id: Optional[str] = None, max_history_tokens: int = 200):
+    def __init__(
+        self, history: List[Message] = [], id: Optional[str] = None, max_history_tokens: int = 200, model: str = "gpt2"
+    ):
```

Comment:
If `max_history_tokens` is set to None, and default should not be 200 — but None — do no trimming and let the damn thing error out.